contents/docs/cdp/destinations/index.md

```diff
@@ -5,132 +5,68 @@ showTitle: true
-
-```rust
-let res := fetch(inputs.url, {
-  'headers': inputs.headers,
-  'body': inputs.body,
-  'method': inputs.method
-});
-
-if (res.status >= 400) {
-  print('Bad response', res.status, res.body)
-}
-```
-
-### Guidelines
-
-- **Start from an existing template:** Check out existing templates that are close to your kind of use case and work from there.
-
-- **Rely on filters:** Offload as much as possible to the built in `filters` and `inputs`. This will make modifying your destination later much simpler (as well as being more performant)
-
-- **Use `inputs` wherever possible:** These are great for secrets such as API credentials. You can mark them as secret and they will not be returned to the UI in the future and will be encrypted.
-
-- **Keep it short:** We have tight controls on execution time, memory usage etc.
-
-- **Do not do more than 2 `fetch` calls:** The function will error if you do. If you have a need for more than 2 calls, please [contact support](https://us.posthog.com/#panel=support%3Asupport%3Aapps%3A%3Atrue).
-
-## FAQ
+## Using destinations
+
+As PostHog data arrives, you can export it *immediately* to other tools. You can use this for things like:
+
+- Conversion tracking
+- Monitoring key events in a chat platform
+- Enriching customer data in your CRM
+- Triggering automations
+
+PostHog enables you to send realtime data to dozens of pre-configured destinations. Check out the Integrations tab and choose **destinations** to get moving.
+
+![Destinations list in PostHog](https://res.cloudinary.com/dmukukwp6/image/upload/destinations_list_acc7e07ae7.png)
+
+Missing a destination you need? Our [webhook destination]() enables you to send events to any HTTP endpoint, using whatever request method, URL parameter values and JSON payload structure your application requires. You can mix hard-coded keys and values with any person or event data. This integrates PostHog with yet more tools, or even your own internal services.
+
+Wherever your data is going, we’ve built destination templates to get you up and running with just a few minutes of effort.
+
+## Filtering
+
+Instead of the firehose of all your PostHog data, you can construct a query that filters by event types, properties, or any HogQL statement you can come up with, so that only the information you care about is sent to your destination.
+
+![Filtering destinations](https://res.cloudinary.com/dmukukwp6/image/upload/filter_ui_8c7b1fb3be.png)
+
+## Testing
+
+The hardest part of integrating two services is making sure everything works as you expect. Every destination includes a built-in testing interface, enabling you to send real data from PostHog on-demand to your target service and debug any errors.
+
+![Destination testing UI](https://res.cloudinary.com/dmukukwp6/image/upload/destination_testing_ui_3fa41f99f2.png)
```

Comment:
This has an access token in the log... Generally I would _not_ recommend that 😅 even if it is a testing one it doesn't look great to have that in our public docs...

Reply:
VERY reasonable, replacing